pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -75,6 +75,9 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   INDEXING_FAILURES("attributeValues", true),
   QUERY_HAS_MV_SELECTION_ORDER_BY("queryHasMVSelectionOrderBy", false),
 
+  HEALTHCHECK_OK_CALLS("healthcheck", true),
```

Comment:
Should we separate the liveness check and readiness check? Currently we are adding the readiness check meter, so suggest naming them `READINESS_CHECK_OK_CALLS`

Reply:
I think it makes more sense to call it "readiness", since there's nothing wrong with server during its starting time. So I've renamed it to readiness. But I don't fully understand what's the difference between "health" and "liveness".

Reply:
https://github.com/apache/pinot/issues/9023 should explain the details best